lib/steps.js

```diff
@@ -1123,8 +1128,6 @@ After(async function(scenario) {
   if (scenario.result.status !== `PASSED`) {
     if ( session_vars.get_debug() ) {
       console.log( `Non-passed status occurred while running test:`, scenario.result.status );
-      // If there's a page visible (locally), give us time to examine it
-      if ( scope.page ) { await scope.page.waitForTimeout( 60 * 1000 ); }
```

Comment:
There's a 1 min. pause happening elsewhere when there's an error. If we want, we can move it from there to here instead, but we shouldn't have it in both spots, I think. This pause was causing issues where the test would "end", but this would still be going on, so it wasn't really ending.

Reply:
Gotcha, I made this comment early and forgot about it by the time I reached the other `waitForTimeout` you modified, sorry!